### PR TITLE
Updates to What's New

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: docs/whatsnew_1.5.7.html [hunk 1]
```diff
@@ -173,19 +173,28 @@ <h2>
...
                 <li>Admin - Default notification behavior for order update (send email/do not send email/hide update) may now be set</li>
                 <li>Admin - Admins can now login to customer accounts to place orders</li>
                 <li>Admin - Added product-details popover to allow quick visibility into products bought in an order without opening the order</li>
                 <li>Admin - Validation for integer min and max values and email addresses</li>
                 <li>Admin - Add sort order to order statuses so the admin order update dropdown can mirror your business workflow</li>
                 <li>Admin - Improve many configuration setting descriptions</li>
                 <li>Admin - Help buttons now link to new documentation</li>
+                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Admin – Product Copy. Expanded to include copy of metatags and linked categories.</li>
+                <li>Admin - removing need for a separate set of language-specific email templates.</li>
```
Comment (drbyte) [inline docs/whatsnew_1.5.7.html:186]: ? Can you elaborate?

Comment (scottcwilson) [reply docs/whatsnew_1.5.7.html:186]: @torvista Your feature - can you provide some more details?

Comment (torvista) [reply docs/whatsnew_1.5.7.html:186]: "Dear"
was hard-coded in several templates, thus requiring a pack of /es templates.
Not anymore.

Comment (scottcwilson) [reply docs/whatsnew_1.5.7.html]: Updated language to be more specific on this point.

**Comment by drbyte** [comment]
Undecided whether this qualifies as a "Whats New" bullet. It's quite trivial, and is the sort of bullet I'd remove when fine-tuning the list for final release.
But I mention the change here for exposure: https://github.com/zencart/zencart/commit/69509c477bbbc7486ec884a221f5b1878eaea2a1

**Comment by scottcwilson** [comment]
My vote would be to include!  We are improving the administrator experience.

**Comment by scottcwilson** [comment]
AFAIK this list is complete.

File: docs/whatsnew_1.5.7.html [hunk 2]
```diff
@@ -173,19 +173,34 @@ <h2>
...
                 <li>Admin - Admins can now login to customer accounts to place orders</li>
                 <li>Admin - Added product-details popover to allow quick visibility into products bought in an order without opening the order</li>
                 <li>Admin - Validation for integer min and max values and email addresses</li>
                 <li>Admin - Add sort order to order statuses so the admin order update dropdown can mirror your business workflow</li>
                 <li>Admin - Improve many configuration setting descriptions</li>
-                <li>Admin - Help buttons now link to new documentation</li>
+                <li>Admin - On-page help added, with links to new documentation</li>
+                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Admin – Product Copy. Expanded to include copy of metatags and linked categories.</li>
+                <li>Admin - removed "Dear" from email templates to facilitate localization.</li>
```
Comment (drbyte) [inline docs/whatsnew_1.5.7.html:186]: This `Dear` salutation change should probably be moved to the Email group, not Admin.

File: docs/whatsnew_1.5.7.html [hunk 3]
```diff
@@ -173,19 +173,34 @@ <h2>
...
                 <li>Admin - Default notification behavior for order update (send email/do not send email/hide update) may now be set</li>
                 <li>Admin - Admins can now login to customer accounts to place orders</li>
                 <li>Admin - Added product-details popover to allow quick visibility into products bought in an order without opening the order</li>
                 <li>Admin - Validation for integer min and max values and email addresses</li>
                 <li>Admin - Add sort order to order statuses so the admin order update dropdown can mirror your business workflow</li>
                 <li>Admin - Improve many configuration setting descriptions</li>
-                <li>Admin - Help buttons now link to new documentation</li>
+                <li>Admin - On-page help added, with links to new documentation</li>
+                <li>Admin – Products to Categories Link Manager. Addition of global tool to copy linked categories to another product.</li>
+                <li>Admin – Product Copy. Expanded to include copy of metatags and linked categories.</li>
```
Comment (drbyte) [inline docs/whatsnew_1.5.7.html:185]: `Product Copy. Expanded`
perhaps make into one sentence:
`Product Copy expanded`

File: docs/whatsnew_1.5.7.html (hunk 2 above)
Comment (drbyte) [reply docs/whatsnew_1.5.7.html:186]: Actually, it already is, in the "hard-coded text removed" line.
Let's use one or the other, not both.

File: docs/whatsnew_1.5.7.html [hunk 4]
```diff
@@ -173,19 +173,34 @@ <h2>
...
                 <li>Storefront - Product page now includes Ask a Question</li>
                 <li>Storefront - Contact Us page now includes a telephone number</li>
-                <li>Email - SPAM prevention built in to forms</li>
+                <li>Storefront - Improved handling of product restrictions such as units and minimum quantities</li> 
+                <li>Storefront - Define page added to shopping cart page for static content</li> 
+                <li>Email - SPAM prevention built in to forms (Credits @davewest for zenNonCAPTCHA)</li>
+                <li>Email - Improvements made to increase email deliverability</li>
+                <li>Email – hard-coded text removed.
                 <li>Checkout - Updated Square Payment module.  Start taking credit card payments in just 5 minutes! See www.zen-cart.com/partners/square_setup </li>
+                <li>Checkout - layout issues in responsive_classic template fixed.</li>
```
Comment (drbyte) [inline docs/whatsnew_1.5.7.html:202]: These were mobile-only layout changes, specific to order-totals. Let's be clear about that.

Comment (scottcwilson) [reply docs/whatsnew_1.5.7.html]: Changed, but this is not entirely true. #3094 also corrects the shipping page in mobile.  And  #2856 fixes the payment page on the desktop

**Comment by scottcwilson** [comment]
This branch is really ancient.  Please commit it so I can create a new one that's more up to date.

**Comment by drbyte** [comment]
I'd thought the plan was to keep updating this one progressively.

**Pull request merged**